website/pages/docs/deployment.md

```diff
@@ -125,6 +125,76 @@ Then run the command `now` in your project, you will be given a `now.sh` URL as
 
 Check out [Now for GitHub](https://zeit.co/docs/v2/integrations/now-for-github) and [Now for GitLab](https://zeit.co/docs/v2/integrations/now-for-gitlab/) for continuous intergration.
 
+## [Firebase](https://firebase.google.com/)
+
+To host and deploy your site to Firebase.
+
+### Step 1: Create a new web app and select "Also setup Firebase Hosting for this app"
+
+### Step 2: Add the required Firebase scripts to the body of your site
```

Comment:
Maybe just remove the step, you don't really the Firebase scripts for a static site.

Reply:
Firebase provides analytics with those scripts, if you remove the step then there isn't really much point in having any Firebase instructions.